docs/blueprints/Components_Overview.md

```diff
@@ -0,0 +1,73 @@
+Components Overview
+===================
+
+## Components
+
+| Component | DRTM        | Description
+| :-------: | :--:        | :---------:
+| ACM       | TXT         | DCE for TXT made and signed by Intel
+| GRUB2     | TXT, SKINIT | Secure Launch enabled bootloader
+| Linux     | TXT         | Secure Launch enabled Unix-like kernel
+| SKL       | SKINIT      | Free and open source DCE for AMD SKINIT
+| SLRT      | TXT, SKINIT | Data format for sharing information among components
+| Xen       | TXT, SKINIT | Secure Launch enabled type-1 hypervisor
```

Comment:
If these were made into links, this file could easily replace `index.md`.

Reply:
Well, not exactly.  There are two blueprints related to GRUB, no for ACM and "Measured Secure Boot" page doesn't fit it.  I've moved the content, but kept auto-generated links at the top.

https://github.com/TrenchBoot/TrenchBoot.github.io/compare/97941c079ed7b202e52b9bc265febac7b8afd04f..1a18d56ac6cad3dcb05e6dddf5ba68cdc918abfd#diff-7453be7cde9748642eac972011134f410b4caf06c8fa323d4b48d03301868f2a